src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
```

Comment:
I thick there should be "<=" , because the 3rd day should be included, check it please